testProjectAndroidLibrary/build.gradle

```diff
@@ -0,0 +1,97 @@
+// This build is not a complete project, but is used to generate a project.
+// See: ProtobufPluginTestHelper.groovy
+buildscript {
+    repositories {
+        maven { url 'https://maven.google.com' }
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+}
+
+plugins {
+    id 'com.android.library'
+    id 'com.google.protobuf'
+}
+
+repositories {
+    maven { url 'https://maven.google.com' }
+    maven { url "https://plugins.gradle.org/m2/" }
+}
+
+
+android {
+    compileSdkVersion 26
+    buildToolsVersion "26.0.1"
+
+    defaultConfig {
+        minSdkVersion 7
+        targetSdkVersion 23
+        versionCode 1
+        versionName "1.0"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_7
+        targetCompatibility = JavaVersion.VERSION_1_7
+    }
+
+    // https://github.com/square/okio/issues/58
+    lintOptions {
+        warning 'InvalidPackage'
+        abortOnError false
+    }
+
+    dexOptions {
+        javaMaxHeapSize "1g"
+        threadCount 1 // reduce predex thread count to limit memory usage
+    }
+}
+
+protobuf {
+    protoc {
+        artifact = 'com.google.protobuf:protoc:3.0.0'
+    }
+    plugins {
+        grpc {
+            artifact = 'io.grpc:protoc-gen-grpc-java:1.0.0-pre2'
+        }
+        javalite {
+            artifact = 'com.google.protobuf:protoc-gen-javalite:3.0.0'
+        }
+    }
+    generateProtoTasks {
+        all()*.plugins {
+            javalite { }
+        }
+        ofNonTest()*.plugins {
+            grpc {
+                // Options added to --grpc_out
+                option 'lite'
+            }
+        }
+    }
+}
+
+dependencies {
+    compile 'com.android.support:appcompat-v7:23.4.0'
+    compile 'com.squareup.okhttp:okhttp:2.7.5'
```

Comment:
Seems we do not need these dependencies here. It would be better to include only things we need.